core/frontend/src/tile/map/MapLayerImageryProvider.ts

```diff
@@ -65,7 +65,10 @@ export abstract class MapLayerImageryProvider {
   /** @internal */
   protected _firstRequestPromise: Promise<void>|undefined;
 
-  /** @internal */
+  /**
+   * The status of the map layer imagery provider.
+   * @beta
```

Comment:
```suggestion
   * @public @preview
```

Unless you have some reason to think we will make breaking changes to this API before iTwin.js 6.0?